### ISSUE TITLE
Validator runs punishing devs

### ISSUE THREAD

**Issue by stes-acc** [w3c/aria#1997]
ARIA versioning: [role and property deprecation](https://github.com/w3c/aria/issues/1990) vs. removal discussion
Examples: no more group role in list allowed in ARIA 1.2, possible [aria-haspopup](https://github.com/w3c/aria/issues/1024) extended values deprecation (?!)

->Expectation statement regarding versioned Validator checks missing by the ARIA group

Recommended procedures for [unique labels](https://dequeuniversity.com/rules/axe/4.4/duplicate-id-aria#:~:text=The%20value%20assigned%20to%20an,differentiate%20each%20element%20from%20another.) of repetitive content (such as “Details” links in multiple table rows) 

-> Expectation statement regarding unique virtual listings of labels in screen readers missing by the ARIA group

**Comment by MelSumner** [comment]
+1 for discussing this.

**Comment by stes-acc** [comment]
Decided to split this into two different issues:

ARIA versioning: https://github.com/w3c/aria/issues/1990 vs. removal discussion

See https://github.com/w3c/aria/issues/2005.

Recommended procedures for [unique labels](https://dequeuniversity.com/rules/axe/4.4/duplicate-id-aria#:~:text=The%20value%20assigned%20to%20an,differentiate%20each%20element%20from%20another.) of repetitive content

See https://github.com/w3c/aria/issues/2006

**Comment by spectranaut** [comment]
because new issues were opened, closing this one
